src/validator.ts

```diff
@@ -209,19 +179,34 @@ function validateWorkingInput(passedOptions?: Partial<ValidateOptions>) {
 
     }
 
+    // Check Transform
+    if (workingInput.hasOwnProperty(['Transform'])) {
+      // assign defined transforms to the validator
+      workingInputTransform = workingInput['Transform'];
+      // initialize transform output
+      if (!!~workingInputTransform.indexOf('AWS::Serverless-2016-10-31')) {
+          workingInput['SAMOutput'] = {}
+      }
+    }
+
 
     // TODO: Check keys for parameter are valid, ex. MinValue/MaxValue
 
+    // Apply specification overrides
+    applySpecificationOverrides();
+
+    // Process SAM Globals section
+    processSAMGlobals();
+
+    // Apply template overrides
+    applyTemplateOverrides();
 
     // Check parameters and assign outputs
     assignParametersOutput(options.guessParameters);
 
     // Evaluate Conditions
     assignConditionsOutputs();
 
-    // Apply resource specification overrides
```

Comment:
Can this comment be merged into where this is now applied?

Reply:
Does this still apply? ...